cuml/dbscan_demo.ipynb

```diff
@@ -64,17 +66,27 @@
    "source": [
     "## Generate Data\n",
     "\n",
-    "### Host"
+    "### Host\n",
+    "\n",
+    "Generate data in host (CPU) memory using sklearn's dataset functions."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "float32\n"
+     ]
+    }
+   ],
    "source": [
     "host_data, host_labels = make_blobs(\n",
-    "   n_samples=n_samples, n_features=n_features, centers=5, random_state=7)\n",
+    "   n_samples=n_samples, n_features=n_features, centers=5, cluster_std=0.1)\n",
```

Comment:
Why no more random seed?

Reply:
This is what I get for being lazy and attempting to do a merge through the github UI. Fixed.